lib/common/tests/rules/pcmk__evaluate_attr_expression_test.c

```diff
@@ -660,6 +661,18 @@ op_eq_passes(void **state)
     assert_attr_expression(EXPR_OP_EQ_PASSES, pcmk_rc_ok);
 }
 
+#define EXPR_EMPTY_EQ_EMPTY_PASSES                      \
+        "<" PCMK_XE_EXPRESSION " " PCMK_XA_ID "='e' "   \
+        PCMK_XA_ATTRIBUTE "='empty' "                   \
+        PCMK_XA_OPERATION "='" PCMK_VALUE_EQ "' "       \
+        PCMK_XA_VALUE "='' />"
+
+static void
+op_empty_eq_empty_passes(void **state)
```

Comment:
we've been using `op_<op>_<description>` for the test naming, doesn't matter much